### ISSUE TITLE
WASM build fails

### ISSUE THREAD

**Issue by aratama** [Lommix/bevy_aseprite_ultra#17]
Since bevy_aseprite_ultra 0.3, the `basis-universal` feature has been added,
causing wasm builds to fail with the following errors:

```
The following warnings were emitted during compilation:

warning: basis-universal-sys@0.3.1: In file included from vendor/basis_universal/encoder/pvpngreader.cpp:14:
warning: basis-universal-sys@0.3.1: vendor/basis_universal/encoder/../transcoder/basisu.h:53:10: fatal error: 'stdlib.h' file not found
warning: basis-universal-sys@0.3.1: #include <stdlib.h>
warning: basis-universal-sys@0.3.1:          ^~~~~~~~~~
warning: basis-universal-sys@0.3.1: 1 error generated.

error: failed to run custom build command for `basis-universal-sys v0.3.1`

Caused by:
  process didn't exit successfully: `/home/cubbit/magiaforge/target/debug/build/basis-universal-sys-93d679b54a707c99/build-script-build` (exit status: 1)

...

 cargo:warning=In file included from vendor/basis_universal/encoder/pvpngreader.cpp:14:
  cargo:warning=vendor/basis_universal/encoder/../transcoder/basisu.h:53:10: fatal error: 'stdlib.h' file not found
  cargo:warning=#include <stdlib.h>
  cargo:warning=         ^~~~~~~~~~
  cargo:warning=1 error generated.
```

It seems that `basis-universal` doesn't work with wasm:
https://github.com/aclysma/basis-universal-rs/pull/11#issuecomment-1207389804

I'm not sure why bevy_aseprite_ultra requires the `basis-universal`.
I removed the `basis-universal` from `Cargo.toml`, and the build succeeded and worked properly.

**Comment by Lommix** [comment]
When migrating to bevy 0.15 the basic-universal feature included the texture compression functions. I assumed this was the way to go without looking any further into it.

I replaced it with the "png" feature and published 0.3.2

Test it again, let me know.

**Comment by aratama** [comment]
It worked fine. 
Thank you so much.
